common/openapi.yml

```diff
@@ -358,6 +377,24 @@ components:
       description: The storage system for this dataset
       enum: ['tdr', 'wks', 'ext']
 
+    WorkspaceId:
+      description: The workspace UUID
+      type: object
+      required: [ id ]
+      properties:
+        id:
+          type: string
+          format: uuid
```

Comment:
I was thinking of strongly typing the WorkspaceId, so it couldn't accidentally be defined as another type of uuid.